CMakeLists.txt

```diff
@@ -89,6 +103,7 @@ else ()
             )
 
     find_package(OpenCV REQUIRED)
+    find_package(gazebo REQUIRED)
```

Comment:
The find_package call shouldn't be in the if-else block. If it needs to be called individually for OpenCV and gazebo to work then we should probably put it in a macro and call it below in the package finding section.